packages/gitbook/src/components/Search/SearchResults.tsx

```diff
@@ -320,3 +328,70 @@ function withQuestionResult(results: ResultType[], query: string): ResultType[]
 
     return [{ type: 'question', id: 'question', query }, ...(without ?? [])];
 }
+
+export const SearchResultItem = React.forwardRef(function SearchResultItem(
```

Comment:
It looks like you have a circular import. This should be in a different file if it's being imported by packages/gitbook/src/components/Search/SearchPageResultItem.tsx